packages/did-provider-jwk/src/jwk-did-provider.ts

```diff
@@ -0,0 +1,107 @@
+import { IIdentifier, IKey, IService, IAgentContext, IKeyManager } from '@veramo/core-types'
+import { AbstractIdentifierProvider } from '@veramo/did-manager'
+import { VerificationMethod } from 'did-resolver'
+import type { JwkDidSupportedKeyTypes } from '.'

```

Comment:
Perhaps it's easier to declare the type here, or in its own file instead of in the index file. There's less potential for circular dependency issues that way.

Reply:
I have moved the type declarations to a separate file.